runtime/src/epoch_rewards_hasher.rs

```diff
@@ -6,21 +6,21 @@ use {
 };
 
 #[derive(Debug, Clone)]
-pub(crate) struct EpochRewardsHasher {
+pub struct EpochRewardsHasher {
```

Comment:
seems like this module should live in sdk?

Reply:
Yes, probably. Makes sense to me, anyway.
I guess since this PR would make it public, I should do the move here. Will push shortly.

Reply:
could move it in a prelim

Reply:
https://github.com/solana-labs/solana/pull/34934